src/state/megaStore.tsx

```diff
@@ -12,6 +12,7 @@ import {
 import { createStore } from "solid-js/store";
 import { useSearchParams } from "solid-start";
 
+import { showToast } from "~/components";
```

Comment:
this store is kind of outside the "UI tree" so while it might work I'm not super comfy doing ui stuff like show toast from here. if there's a serious error it should set an error for the whole app, if it's not serious it should just fail gracefully

Reply:
Ok! At the moment it shows the toast and immediately switches to BTC as the pair